app/config/questions.py

```
@@ -0,0 +1,208 @@
+    20: {
+        "text": "**Información adicional**\n\n¿Algo más que quieras contarnos?\n\n*(Opcional - Cualquier información adicional que consideres relevante)*",
+        "type": "optional",
+        "required": False,
+        "validation": "optional_text",
+        "rubrica_key": "pregunta_20",
+        "field_name": "additional_info"
+    }
+}
+
+
+def get_question(question_number: int) -> Dict[str, Any]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)